core-bundle/src/Command/SendMailCommand.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+/**
+ * A console command to send an email via Symfony Mailer.
+ */
+final class SendMailCommand extends Command
+{
+    protected static $defaultName = 'contao:sendmail';
+
+    private SymfonyStyle $io;
+    private MailerInterface $mailer;
+
+    public function __construct(MailerInterface $mailer)
+    {
+        parent::__construct();
+
+        $this->mailer = $mailer;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Sends an email message')
```

Comment:
you can leverage https://symfony.com/blog/new-in-symfony-5-3-lazy-command-description